src/uiHelpers_nanox.c

```diff
@@ -50,13 +50,18 @@ UX_FLOW(
 
 void ui_displayPaginatedText_run()
 {
+        ux_flow_init(0, ux_short_text_flow, NULL);
+        #ifdef FUZZING
+        ux_stack_push();
+        #endif
+        /*
 	if (strlen((const char*) &displayState.paginatedText.fullText) < 18 ) {
 		ux_flow_init(0, ux_short_text_flow, NULL);
 	} else {
 		ux_layout_bnnn_paging_reset();
 		ux_flow_init(0, ux_paginated_text_flow, NULL);
 
-	}
+	}*/
```

Comment:
Does your change imply that this code is no longer relevant, i.e. here is not need to call `ux_layout_bnnn_paging_reset` (maybe there never was a need before, I don't know what the function does)? 

Perhaps it could be deleted then?

Reply:
I'm unsure why this comment was necessary at first actually, so I uncommented. Thanks for your review